.gitignore

```diff
@@ -29,6 +29,10 @@ app/release
 localeapi/gradle/
 localeapi/lint.xml
 
+gradle/wrapper/dists
```

Comment:
Which build environment do you use? I've never seen these directories and wonder why they are created.